conf/app.src

```diff
@@ -1,6 +1,6 @@
-SOURCE_URL=https://repository.yeswiki.net/doryphore/yeswiki-doryphore-2021-05-18-2.zip
-SOURCE_SUM=1d22ab67b1d19433b85c81b58483fbb766edea04bfe5e588d489d2a1d0043404
-SOURCE_SUM_PRG=sha256sum
+SOURCE_URL=https://repository.yeswiki.net/doryphore/yeswiki-doryphore-2021-07-16-1.zip
+SOURCE_SUM=96b79805f130a0759fbd1560a15904ec
+SOURCE_SUM_PRG=md5sum
```

Comment:
We should use sha256sum for the shasum.

Reply:
Ok, i will ping you when the change is effective. For now the bot will still commit the changes, with md5sum, but i will implement sha256sum soon i hope

Reply:
How does the bot works? Is it depending on a md5 generated  upstream?

Reply:
maybe bot was not the appropriate term for it, it's a repository builder, that generates zip files and md5sum (for now, because YesWiki's update manager still relies on md5), that also automates the changes for the yunohost package and commits to the yunohost repo.
the source code is there : https://github.com/YesWiki/yeswiki-build-repo

Reply:
I will implemente the feature to also generate sha256sum with this builder, just i'm not sure when ;)

Reply:
```suggestion
SOURCE_SUM_PRG=sha256sum
```